parser/ast/dml.go

```diff
@@ -1042,6 +1042,17 @@ type CommonTableExpression struct {
 	Name        model.CIStr
 	Query       *SubqueryExpr
 	ColNameList []model.CIStr
+	IsRecursive bool
+}
+
+// Restore implement it Before using
+func (c CommonTableExpression) Restore(ctx *format.RestoreCtx) error {
+	panic("implement me")
+}
+
+// Accept implement it Before using
+func (c CommonTableExpression) Accept(v Visitor) (node Node, ok bool) {
+	panic("implement me")
```

Comment:
v.Enter(cte)
		node, ok := cte.Query.Accept(v)
		if !ok {
			return n, false
		}
		cte.Query = node.(*SubqueryExpr)
		v.Leave(cte)

Reply:
cte.query is `SubqueryExpr`. I dont find the usage of CommonTableExpression `Accept ` and `store`. So I just panic to prevent someone from using it without implementing it during development

Reply:
cte.Query.Accept(v) -> cte.Accept(v)